src/ui/handler/drag_pan.js

```diff
@@ -147,28 +154,39 @@ class DragPanHandler {
 
         this._state = 'pending';
         this._startPos = this._mouseDownPos = this._prevPos = this._lastPos = DOM.mousePos(this._el, e);
+        this._startTouch = this._lastTouch = e instanceof window.TouchEvent ? DOM.touchPos(this._el, e) : null;
```

Comment:
import `window`?

Reply:
@arindam1993 maybe I'm misunderstanding what you mean but we import `window` in line 5

Reply:
No you're not, my derp 🤦‍♂